src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -129,6 +129,11 @@ public PublicationController(PublicationService publicationService, ValidationSe
         Artefact createdItem = publicationService
             .createPublication(artefact, payload);
 
+        //TODO handle the below in some way
+        @SuppressWarnings("PMD.PrematureDeclaration")
+        List<Subscription> unusedListOfEligibleSubscribers =
```

Comment:
Just thinking through the scenario's. Is the unused list needed in data management? It feels more like a subscription-management thing

Reply:
sorted.